assets/js/theme/common/utils/banner-utils.js

```diff
@@ -0,0 +1,27 @@
+import { isBrowserIE } from './ie-helpers';
+
+const bannerUtils = {
+    dispatchProductBannerEvent: (productAttributes) => {
+        let price = 0;
+
+        if (!productAttributes.price || isBrowserIE) return;
+
+        if (productAttributes.price.without_tax && !productAttributes.price.price_range) {
+            price = productAttributes.price.without_tax.value;
+        }
+
+        if (productAttributes.price.with_tax && !productAttributes.price.price_range) {
+            price = productAttributes.price.with_tax.value;
+        }
```

Comment:
could we wrap this piece into `!productAttributes.price.price_range if statement`

Reply:
Ok, IMHO to much nested `lfs` reduce readability too, but for this case its doesn't matter